conf/application.conf

```diff
@@ -339,8 +319,6 @@ feature {
       enabled = false
       heading.en = "Help make GOV.UK better"
       link.text.en = "Sign up to take part in research (opens in new tab)"
```

Comment:
Weird. The welsh has been deleted but not the english?

Reply:
I believe this is not supposed to be removed and is removed by error. I will add it back,